recipe/meta.yaml

```diff
@@ -1,45 +1,82 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "raphtory" %}
+{% set version = "0.13.1" %}
+{% set tests_to_skip = "test_add_properties" %}
+{% set tests_to_skip = tests_to_skip + " or test_graph_file_time_stats" %} # [osx and x86_64]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  url: https://github.com/Pometry/Raphtory/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f99caa183387c68a3cdc0949c9000c23553cb3f33524989fa14170a3f522f69d
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  script:
+    - {{ PYTHON }} -m pip install ./python -vv --no-deps --no-build-isolation
+    - cargo-bundle-licenses --format yaml --output "{{ SRC_DIR }}/THIRDPARTY.yml"
+  # gql-with-all not on s390x
+  skip: True  # [py<38 or s390x]
 
 requirements:
+  build:
+    # C Compiler is needed on linux to get missing libgcc-ng as a run_export
+    - {{ compiler('c') }}     # [linux]
+    - {{ compiler('rust') }}
+    - cmake
+    - cargo-bundle-licenses
   host:
     - python
     - pip
+    - protobuf
+    - maturin >=1.4.0
   run:
     - python
+    - pyvis >=0.3.2
+    - networkx >=2.6.3
+    - pandas >=2.0.3
+    - pyarrow >=14.0.1,<16
+    - requests >=2.31.0
+    - gql-with-all ==3.5.0
+    - matplotlib-base >=3.4.3
+    - ipywidgets >=8.1.5
 
```

Comment:
Should we have a `seaborn >= 0.11.2` `run_constrained`?

Reply:
I wouldn't add it since it is marked as `plot` and our distribution is headless. Same rational behind `matplot-lib`.
https://github.com/Pometry/Raphtory/blob/85a9eab8839cc2e602f2737897e48de8653ad967/python/pyproject.toml#L35